webapp/models.py

```diff
@@ -36,6 +37,74 @@ def __init__(self, document, *args, **kwargs):
         self.document = document
 
 
+# Private helper functions
+# ===
+
+
+def _process_html(html):
+    """
+    Post-process the HTML output from Discourse to
+    remove 'NOTE TO EDITORS' sections
+    """
+
+    soup = BeautifulSoup(html, features="html.parser")
+    notes_to_editors_spans = soup.find_all(text="NOTE TO EDITORS")
+
+    soup = _replace_notifications(soup)
+
+    for span in notes_to_editors_spans:
+        container = span.parent.parent.parent.parent
```

Comment:
I think here deserves a comment

Reply:
Even though I didn't actually change this line in this PR, I do agree. We should always err on the side of adding comments whenever anything is unclear.

I've now added comments. I hope they clarify things.